db/migrate/20210616124056_create_users.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
```

Comment:
have a look at devise, which also provides migration generator

Reply:
yep, https://github.com/heartcombo/devise#getting-started
```
rails generate devise User
```
you will need just to edit a bit generated migration to add some custom fields like `first_name`, `last_name` etc